config/environments/test.rb

```diff
@@ -45,6 +45,8 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
+  config.action_mailer.default_url_options = {:host => "localhost:3000"}
```

Comment:
Most people prefer this syntax: `{host: "localhost:3000"}` ;)

Reply:
thanks, corrected. 👍

Reply:
Actually, it's nothing wrong with the previous version - both are correct and doing the same. It's just that it's good to be consistent and stick to one version :)